code/controllers/subsystem/persistence.dm

```diff
@@ -259,6 +260,16 @@ SUBSYSTEM_DEF(persistence)
 	if(CONFIG_GET(flag/use_antag_rep))
 		CollectAntagReputation()
 	SaveRandomizedRecipes()
+	SavePanicBunker()
+
+/datum/controller/subsystem/persistence/proc/LoadPanicBunker()
+	var/bunker_path = file("data/bunker_passthrough.json")
+	if(fexists(bunker_path))
+		GLOB.bunker_passthrough = json_decode(file2text(bunker_path))
+		var/round_id = GLOB.round_id
+		for(var/ckey in GLOB.bunker_passthrough)
+			if(round_id - GLOB.bunker_passthrough[ckey] > 100)
```

Comment:
The panic bunker duration should be in real time for consistency and being easier to manage. And perhaps a configuration instead of a hardcoded number.

Reply:
done